modules/core/src/main/scala/org/scalasteward/core/forge/data/NewPullRequestData.scala

```diff
@@ -66,19 +66,24 @@ object NewPullRequestData {
 
         val updateInfoUrls = artifactIdToUpdateInfoUrls.getOrElse(u.mainArtifactId, Nil)
 
-        s"""|Updates $artifacts ${fromTo(u)}.
-            |${renderUpdateInfoUrls(updateInfoUrls).getOrElse("")}""".stripMargin.trim
+        s"""|## Update _${artifacts}_
```

Comment:
I don't think we can use `${artifacts}` in a header here since it could be a bulleted list as for example in this PR: https://github.com/scala-steward-org/scala-steward/pull/3003

[Here](https://github.com/scala-steward-org/scala-steward/blob/9757397b1e34d2707819aa3dd9f17754f1412bb8/modules/core/src/main/scala/org/scalasteward/core/forge/data/NewPullRequestData.scala#L141-L151) is the function that creates this list in case of an `Update.ForGroupId`.